### PR TITLE
Change all absolute URLs attempting to access public/static contents to use relative URLs

### LINKED ISSUE #5303: Migrate usages of `REACT_PUBLIC_URL` to use `config.json`'s `site_url`

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- Resolves #5303

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Resource/ResourceDetails.tsx [hunk 1]
```diff
@@ -103,10 +103,7 @@ export default function ResourceDetails(props: { id: string }) {
     return (
       <div id="section-to-print" className="print bg-white">
         <div className="flex justify-center">
-          <img
-            src={`${import.meta.env.REACT_PUBLIC_URL}/${KeralaLogo}`}
-            alt="kerala-logo"
-          />
+          <img src={`/${KeralaLogo}`} alt="kerala-logo" />
```
Comment (sainak) [inline src/Components/Resource/ResourceDetails.tsx]: Shouldn't this be replaced with the symbol of the state for which the instance is deployed?

Comment (sainak) [reply src/Components/Resource/ResourceDetails.tsx]: the `KeralaLogo` const is only being used here

Comment (rithviknishad) [reply src/Components/Resource/ResourceDetails.tsx]: @mathew-alex could you comment? This is about the approval letter in the resource section.
image

Comment (rithviknishad) [reply src/Components/Resource/ResourceDetails.tsx]: After discussing it with @nihal467 it's decided to be removed for now. If we need to show `state_logo` it'll be followed up by #5318

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**